packages/indexer-common/src/network-subgraph.ts

```diff
@@ -54,6 +55,10 @@ export class NetworkSubgraph {
   private constructor(options: NetworkSubgraphOptions) {
     this.logger = options.logger
     this.freshnessChecker = options.subgraphFreshnessChecker
+    this.endpoint =
```

Comment:
This will give us a static endpoint value depending on the config, right? 
The client is dynamic though depending on if the local deployment is synced to chain head (switches to the external endpoint if the local deployment is not synced).

Reply:
how do you know what is an external endpoint? I saw that this class either calls the `options.endpoint` or uses the `deployment` subgraph (which I assume is what local deployed to graph-node)

Reply:
options.endpoint is what I mean by "external".  Though you are right it doesn't necessarily mean it's external, it's just a url.